cob_mimic/src/mimic_node.py

```diff
@@ -60,6 +60,7 @@
 import sys
 import os
 import subprocess
+import vlc
```

Comment:
is this python module available via `pip`?
adding the whole modules seems overkill...